sdk/resourcemanager/azure-resourcemanager-containerservice/src/main/java/com/azure/resourcemanager/containerservice/models/KubernetesCluster.java

```diff
@@ -195,7 +195,7 @@ interface WithGroup extends GroupableResource.DefinitionStages.WithGroup<WithVer
         /**
          * The stage of the Kubernetes cluster definition allowing to specify the managed cluster SKU.
          * LongTermSupport and Premium tier should be enabled/disabled together.
-         * For more information, please see https://aka.ms/aks/lts
+         * For more information, please see https://learn.microsoft.com/en-us/azure/aks/long-term-support
```

Comment:
Can we use `https://aka.ms/aks/lts`?

If not, do not use the `en-us` in the link.

Reply:
The `https://aka.ms/aks/lts` is a blog, I think it is better to use `https://learn.microsoft.com`.